great_expectations/checkpoint/checkpoint.py

```diff
@@ -461,3 +494,57 @@ def _get_batches_to_validate(self, batches):
                 batches_to_validate.append(batch)
 
         return batches_to_validate
+
+
+class SimpleCheckpoint:
+    def __init__(
+        self,
+        name: str,
+        data_context,
+        config_version: Optional[Union[int, float]] = None,
+        template_name: Optional[str] = None,
+        module_name: Optional[str] = None,
+        class_name: Optional[str] = None,
+        configurator: Optional[dict] = SimpleCheckpointConfigurator,
+        run_name_template: Optional[str] = None,
+        expectation_suite_name: Optional[str] = None,
+        batch_request: Optional[Union[BatchRequest, dict]] = None,
+        action_list: Optional[List[dict]] = None,
+        evaluation_parameters: Optional[dict] = None,
+        runtime_configuration: Optional[dict] = None,
+        validations: Optional[List[dict]] = None,
+        profilers: Optional[List[dict]] = None,
+        validation_operator_name: Optional[str] = None,
+        batches: Optional[List[dict]] = None,
+    ):
+        super().__init__(
```

Comment:
Add `slack_webhook` here that is given as an option for the configurator (i.e. into parent **kwargs)